addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -1299,7 +1299,7 @@ var FieldFloatToggle = AbstractField.extend({
      * @param {OdooEvent} ev
      */
     _onClick: function(ev) {
-        if (this.mode === 'edit') {
+        if (this.mode === 'edit' || this.nodeOptions.force_button) {
             ev.stopPropagation(); // only stop propagation in edit mode
```

Comment:
You should adapt the comment if your condition is modified.

Reply:
@arj-odoo Done

Reply:
you can delete the comment '// only stop propagation in edit mode'. it is not useful anymore.

Reply:
@arj-odoo done